### ISSUE TITLE
In Manage Edit Topic name /camp  without updating any details just click on submit update button ,its going to review state. 

### ISSUE THREAD

**Issue by sonalizibtek** [the-canonizer/canonizer.2.0#1232]
**Case:**
1.create a topic and camp User A
2.Login with User B ,Click on agreement camp
3.click on manage edit this camp 
4.click on submit camp update based on this 
5.and it will redirect to edit camp page , there don't update anything and click on submit update

**Actual Result :**
1. Without entering details , in camp history page it is showing in review history and commit button  (user B)
2.In User A side , there is a check box , but user B dint enter any details, just he clicked on commit button.
and when User A is reviewing the changes , he is not finding any change 

**Question**:
But when  **User A** review this camp, he wont find any changes?  
@rupalichavan9860 @BrentAllsop , Please confirm this case 

**Expected :**
1.If user dint update any change then it should not show In review state 
<img width="960" alt="Screenshot (135)" src="https://user-images.githubusercontent.com/99309948/157708727-d83683ee-2625-435c-8f4f-0e7e480e54f7.png">
<img width="960" alt="Screenshot (136)" src="https://user-images.githubusercontent.com/99309948/157708767-a1e15f92-e1ed-4f42-bc00-1fbc987e742b.png">

**Comment by rupalichavan9860** [comment]
@sunilspr7 Tested  and working fine now . As per current implementation, button is enabled and non clickable when there are no changes on the form
But I would suggest it will be good if we disabled the button

**Comment by rupalichavan9860** [comment]
@sunilspr7 If user added any change and deleted the same and then button gets enabled and user is able to submit change , record is getting saved as a new record 

But we can implement this in 3.0 
@gautamv16 Let me know your thoughts

**Comment by gautamv16** [comment]
@rupalichavan9860 yes not required now for 2.0. Let's pass this for 2.0 and create ticket for 3.0 to implement same.

**Comment by rupalichavan9860** [comment]
Sure @gautamv16

**Comment by rupalichavan9860** [comment]
> @rupalichavan9860 yes not required now for 2.0. Let's pass this for 2.0 and create ticket for 3.0 to implement same.

One more case : 
Login with User B and try to submit 1st change without updating any field- It should allow because nick name is different, but as of now it is not allowing
I will add this cases for 3.0
